Week9 - homework SQL/LibrariesManagement.sql

```diff
@@ -0,0 +1,298 @@
+VALUES('Lecturarium', 'Emil Racovita', 'Iasi')
+
+SELECT *
+FROM Libraries;
+
+ALTER TABLE Librarians
+ALTER COLUMN IsOnHoliday BIT;
+
+INSERT INTO Librarians
+VALUES('Andreea', '2020-05-10', 0, 1)
+INSERT INTO Librarians
+VALUES('Paul', '2016-07-30', 1, 2)
+INSERT INTO Librarians
+VALUES('Simona', '2021-01-24', 1, 4)
+INSERT INTO Librarians
+VALUES('Traian', '2023-01-15', 0, 4)
+INSERT INTO Librarians
+VALUES('Larisa', '2012-11-30', 0, 3)
+
+SELECT *
+FROM Librarians;
+
+INSERT INTO Books
+VALUES('Prietena mea geniala', 'Elena', 'Ferrante', 'fantasy', 'ferrante@gmail.com')
+INSERT INTO Books
+VALUES('Povestea noului nume', 'Elena', 'Ferrante', 'fantasy', 'ferrante@gmail.com')
+INSERT INTO Books
+VALUES('Scurta Istorie a Pamantului', 'Andrew', 'Knoll', 'nonfiction', 'a_knoll@gmail.com')
+INSERT INTO Books
+VALUES('Nascut in afara legii', 'Trevor', 'Noah', 'memoir', 'trevorrr@gmail.com')
+INSERT INTO Books
+VALUES('Din cer au cazut trei mere', 'Narine', 'Abgarian', 'fantasy', 'abgarian@gmail.com')
+INSERT INTO Books
+VALUES('Cei care pleaca si cei care raman', 'Elena', 'Ferrante', 'fantasy', 'ferrante@gmail.com')
+INSERT INTO Books
+VALUES('Ciuleandra', 'Liviu', 'Rebreanu', 'crime', 'rebreanu@postaromana.ro')
+INSERT INTO Books
+VALUES('Povestea fetitei pierdute', 'Elena', 'Ferrante', 'fantasy', 'ferrante@gmail.com')
+
+INSERT INTO Books_Libraries
+VALUES(1, 1)
+INSERT INTO Books_Libraries
+VALUES(1, 3)
+INSERT INTO Books_Libraries
+VALUES(2, 1)
+INSERT INTO Books_Libraries
+VALUES(2, 3)
+INSERT INTO Books_Libraries
+VALUES(3, 3)
+INSERT INTO Books_Libraries
+VALUES(4, 1)
+INSERT INTO Books_Libraries
+VALUES(5, 4)
+INSERT INTO Books_Libraries
+VALUES(6, 1)
+INSERT INTO Books_Libraries
+VALUES(6, 3)
+INSERT INTO Books_Libraries
+VALUES(7, 4)
+INSERT INTO Books_Libraries
+VALUES(8, 1)
+INSERT INTO Books_Libraries
+VALUES(8, 3)
+
+SELECT *
+FROM Books_Libraries;
+SELECT *
+FROM Libraries;
+SELECT *
+FROM Books;
+
+INSERT INTO Sales
+VALUES(114, 1)
+INSERT INTO Sales
+VALUES(98, 4)
+INSERT INTO Sales
+VALUES(321, 3)
+INSERT INTO Sales
+VALUES(987, 2)
+INSERT INTO Sales
+VALUES(89, 7)
+INSERT INTO Sales
+VALUES(56, 8)
+INSERT INTO Sales
+VALUES(347, 5)
+INSERT INTO Sales
+VALUES(563, 11)
+INSERT INTO Sales
+VALUES(97, 6)
+INSERT INTO Sales
+VALUES(224, 9)
+INSERT INTO Sales
+VALUES(90, 12)
+INSERT INTO Sales
+VALUES(542, 10)
+
+SELECT *
+FROM Sales;
+
+--exercise2
+--Display all the librarians together with the library name they work in ordered by librarian name and library name descending.
+SELECT L.Name, Lib.Name
+FROM Librarians L
+INNER JOIN Libraries Lib ON Lib.Id = L.LibraryId
+ORDER BY L.Name DESC;
+
+SELECT L.Name, Lib.Name
+FROM Librarians L
+INNER JOIN Libraries Lib ON Lib.Id = L.LibraryId
+ORDER BY Lib.Name DESC;
+
+--exercise3
+--Update all the street names to start with "STR.".
+UPDATE Libraries
+SET StreetName = 'STR. ' + Streetname;   --I didn't know how to undo the 1st update. That's why the 1st try of 'str' is still there
```

Comment:
I can't see the first try because I'm running the script for the first time so I'm working on a "clean" database.